### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by MudasirSherwani**
Add Work Section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
...
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-4">
```
Comment (victorokoroji) [inline index.html:165]: - [ ] Good job using flexbox to align your project contents 👍🏼 . Your project cards are currently not centralized or centered on the page as they are all aligned to the left. Kindly align the project elements to the center with equal margin to match the Figma design template you chose 😄 

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
...
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
```
Comment (victorokoroji) [inline index.html]: - [ ] Kindly ensure you centralize your list of technologies with the appropriate margin as specified in the Figma template you chose👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @MudasirSherwani, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @MudasirSherwani  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**